src/generators/man-page/index.mjs

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import { writeFile, readFile } from 'node:fs/promises';
+import { join } from 'node:path';
+
+import {
+  convertOptionToMandoc,
+  convertEnvVarToMandoc,
+} from './utils/converter.mjs';
+
+/**
+ * This generator generates a man page version of the CLI.md file.
+ * See https://man.openbsd.org/mdoc.7 for the formatting.
+ *
+ * @typedef {Array<ApiDocMetadataEntry>} Input
+ *
+ * @type {import('../types.d.ts').GeneratorMetadata<Input, string>}
+ */
+export default {
+  name: 'man-page',
+
+  version: '1.0.0',
+
+  description: 'Generates the Node.js man-page.',
+
+  dependsOn: 'ast',
+
+  async generate(input, options) {
+    // Find the appropriate headers
+    const optionsStart = input.findIndex(({ slug }) => slug === 'options');
+    const environmentStart = input.findIndex(
+      ({ slug }) => slug === 'environment-variables-1'
+    );
+
+    if (optionsStart + environmentStart <= 0) {
```

Comment:
This should never happen, right? Then I don't think this Error should be thrown (IMO)

Reply:
It'll happen if the user passes a file that isn't CLI.md.

Reply:
What if they pass a bunch of files, are you able to filter for the cli.md?

Check if any node with the slug or name of cli is being passed :)

Reply:
And then throw error -- because other nodes/files could have the heading of options